SaveMyEyes/sp.lproj/Localizable.strings

```diff
@@ -0,0 +1,23 @@
+/* 
+ Localizable.strings
+ SaveMyEyes
+ 
+ Creado por Max Omelchenko el 24/02/20.
+ Copyright © 2020 Max Omelchenko. Todos los derechos reservados.
+ */
+
+"Time to work" = "Hora de trabajar";
+"Time to break" = "Hora de pausa";
+"Run timer" = "Ejecutar Temporizador";
+"min" = "min";
+"Work interval" = "Intervalo de trabajo";
+"Break interval" = "Intervalo de pausa";
+"Quit" = "Salir";
+"It's time for break" = "Es hora del descanso";
+"It's time to work" = "Es hora de trabajar";
+"Relax from your computer for %d min." = "Relájate de tu ordenador durante %d min.";
+"Let's continue to do amazing things!" = "¡Sigamos haciendo cosas increíbles!";
+“Pause” = "Pausa";
```

Comment:
Could you please replace `”` with `"` here as well?

Reply:
@masich Done!